templates/party/edit.html.twig

```diff
@@ -0,0 +1,54 @@
+{% extends 'layout-with-submenu.html.twig' %}
+
```

Comment:
Does this need a translation domain?

Reply:
Added translation domain and made translations